### PR TITLE
Add support for multiple schema directories

### PR DESCRIPTION
**Pull request by bnatrajan**

### CODE REVIEW

File: rde_schema_dictionary_gen.py [hunk 1]
```diff
@@ -1051,24 +1074,38 @@ def print_binary_dictionary(byte_array):
 
     if args.source == 'local':
         entity = args.entity
-        source = args.schemaDir + '/' + 'metadata/' + args.schemaFilename
+
+        # compute source starting with the first csdl directory. The first one wins
+        source = args.schemaFilename
+        for csdl_dir in args.csdlSchemaDirectories:
+            if os.path.isfile(csdl_dir + '/' + args.schemaFilename):
```
Comment (adkarira) [inline rde_schema_dictionary_gen.py]: Should we handle the case where csdl_dir already contains a trailing '/'?

File: rde_schema_dictionary_gen.py [hunk 2]
```diff
@@ -294,6 +295,20 @@ def add_actions(doc, entity_repo):
...
+def find_csdl_source(filename):
+    for csdl_dir in args.csdlSchemaDirectories:
+        if os.path.isfile(csdl_dir + '/' + filename):
+            return csdl_dir + '/' + filename
+    return ''
+
+
+def find_json_schema_source(filename):
+    for json_schema_dir in args.jsonSchemaDirectories:
+        if os.path.isfile(json_schema_dir + '/' + filename):
```
Comment (adkarira) [inline rde_schema_dictionary_gen.py]: Should we handle the case where json_schem_dir already contains a trailing '/'?

File: rde_schema_dictionary_gen.py (hunk 1 above)
Comment (bnatrajan) [reply rde_schema_dictionary_gen.py]: should be ok to have 2 '//', the system will handle it as a single one

File: rde_schema_dictionary_gen.py (hunk 2 above)
Comment (bnatrajan) [reply rde_schema_dictionary_gen.py]: should be ok to have 2 '//', the system will handle it as a single one

**Pull request merged**